example/bin/ci/example.sh

```diff
@@ -21,7 +21,7 @@
 cd "$(dirname "$0")" || exit 1
 
 # Reuse ORCA's own includes for its $PATH additions and environment variables.
-source ../../../orca/bin/ci/_includes.sh
+source ../../../orca/bin/ci/_includes.sh || exit
```

Comment:
We should go the rest of the way and explicitly emit an error code:

```suggestion
source ../../../orca/bin/ci/_includes.sh || exit 1
```

Reply:
Nit, but I don't think an explicit error code is necessary, since `exit` returns the exit code from the previous command, which in this case is `source`. But it doesn't really matter unless ORCA has a set of explicitly-defined exit codes that I'm not aware of 😄